### PR TITLE
Add missed profile and mutator keys to the validation schema.json

### PR DESCRIPTION
**Pull request by sidz**
This PR adds missed profile and mutator keys to the validation schema.json

Requested here: https://github.com/infection/infection/issues/732
Missed here: https://github.com/infection/infection/pull/524

### CODE REVIEW

File: resources/schema.json [hunk 1]
```diff
@@ -92,6 +92,994 @@
             "type": "object",
             "description": "Contains the settings for different mutations and profiles",
             "properties": {
+                "@arithmetic": {
+                    "anyOf": [
+                        {
+                            "type": "boolean"
+                        },
+                        {
+                            "type": "object",
```
Comment (theofidry) [inline resources/schema.json]: oh, this is valid for profiles as well?

**Comment by theofidry** [comment]
@sidz might be worth taking the `schema.json` from https://github.com/infection/infection/pull/750, there is a little trick there to remove the redundancy of declaring a schema over and over

**Comment by sidz** [comment]
@theofidry didn't find about which trick are you talking. Could you please point what where it is?

**Comment by theofidry** [comment]
Sorry I actually didn't push that part, done now: https://github.com/infection/infection/pull/750/files#diff-29ca588982da1e8d8324a8f6cf78ce34R3

**Comment by sidz** [comment]
@theofidry cheers. applied

File: resources/schema.json [hunk 2]
```diff
@@ -281,5 +448,39 @@
...
+            "anyOf": [
+                {
+                    "type": "boolean"
+                },
+                {
+                    "$ref": "#/definitions/ignore-option"
+                }
+            ]
+        },
+        "mutator": {
```
Comment (maks-rafalko) [inline resources/schema.json]: seems like unused and exactly the same as `mutator_profile`

**Comment by sanmai** [comment]
I wonder if we should have a test for this list, to check that all mutators are indeed present.

**Comment by maks-rafalko** [comment]
Thank you @sidz

**Pull request merged**